src/main/java/core/basesyntax/Factorial.java

```diff
@@ -4,8 +4,14 @@
  * <p>Реализуйте метод getFactorial(int number) который вовзращает факториал числа.
  * Реализовать с помощью рекурсии.</p>
  */
+
 public class Factorial {
     public int getFactorial(int number) {
-        return 0;
+        int result = 1;
+        if (number == 1 || number == 0) {
```

Comment:
let's simplify this to one condition

Reply:
If I understood correctly you ask to rewrite with the ternary expressions and to use one return!?